addons/gatekeeper/gatekeeper.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: gatekeeper
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "3.3.0-6"
-    appversion.kubeaddons.mesosphere.io/gatekeeper: "3.3.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "v3.4.0-rc.1-1"
```

Comment:
We should not be tying addon (nor chart) versions to app versions.

Reply:
i think this is the intent of `catalog.kubeaddons.mesosphere.io/addon-revision`. it's so that the catalog can show the app version and sort accordingly.

Reply:
maybe it should be `3.4.0-1` instead?